packages/block-editor/src/components/block-list/block-mobile-floating-toolbar.native.scss

```diff
@@ -1,4 +1,5 @@
-$floating-toolbar-height: 44;
+$floating-toolbar-height: 44px;
+$floating-toolbar-shift: 47px;
```

Comment:
why is this 47? is it 44 + margins maybe? if so we should calculate it as $floating-toolbar-height+$block-selected-margin

Reply:
When I get done cleaning and refining I'll adjust so we do something like you suggested. I need to investigate how these numbers are being translated and check them on iOS. The goal is to have the Floatingtool bar 8 px above the block. For some reason, `$floating-toolbar-height + 3px` seems to be the magic number to get to 8 when I use the layout inspector. Not sure why that is the case yet though. I need to look into it more.